sdk/keyvault/keyvault-certificates/CHANGELOG.md

```diff
@@ -1,5 +1,20 @@
 # Release History
 
+## 4.0.0-preview.9 (2019-11-04)
+
+- Fixed parameter names and access modifiers to be on par of other language implementations.
```

Comment:
I believe we can skip this one too.
Changes to parameter names to methods don't have a net affect on the final end user